SPECS/kernel/CVE-2023-22995.nopatch

```diff
@@ -0,0 +1,3 @@
+CVE-2023-22995 - patched in 5.15.10.2.1
```

Comment:
Nit. I think there is an extra dot :)

Reply:
Thanks! I remember it looked weird. Fixing
![image](https://github.com/microsoft/CBL-Mariner/assets/30801614/b55b1830-ac50-41eb-a6a4-b5213278dcec)

Reply:
Shouldn't it be from 5.15 instead of 5.10, or it doesn't matter here since stable and mainline have the same commit hash?